Kickstarter-iOS/Views/Cells/DiscoveryPostcardCell.swift

```diff
@@ -189,6 +189,7 @@ internal final class DiscoveryPostcardCell: UITableViewCell, ValueCell {
     self.viewModel.outputs.metadataData
       .observeForUI()
       .observeValues { [weak self] data in
+        self?.metadataIconImageView.isHidden = data.iconImage == nil
```

Comment:
@justinswart, @theginbin,  I noticed it too. I don't see problem with this approach but it is different from our "regular way" of doing things. I would be very interested to have a short discussion on why having only one side-effect per signal is a better approach..